### ISSUE TITLE
Requests only sent with `-n 50` or higher

### ISSUE THREAD

**Issue by howardjohn** [bojand/ghz#233]
it seems any value of `-n` less than 50 is not sending requests:

```
$ ghz --skipTLS localhost:15012 --call envoy.service.discovery.v3.AggregatedDiscoveryService/StreamAggregatedResources -n 49

Summary:
  Count:        0
  Total:        0.94 ms
  Slowest:      0 ns
  Fastest:      0 ns
  Average:      0 ns
  Requests/sec: 0.00

Response time histogram:

Latency distribution:



$ ghz --skipTLS localhost:15012 --call envoy.service.discovery.v3.AggregatedDiscoveryService/StreamAggregatedResources -n 50

Summary:
  Count:        50
  Total:        45.67 ms
  Slowest:      0 ns
  Fastest:      0 ns
  Average:      24.82 ms
  Requests/sec: 1094.81

Response time histogram:

Latency distribution:

Status code distribution:
  [Unknown]   50 responses

Error distribution:
  [50]   rpc error: code = Unknown desc = authentication failure
```

The server is a bidi streaming service

**Comment by howardjohn** [comment]
Likely matching the concurrency option of `50`

**Comment by bojand** [comment]
Hello, thanks for bringing up the issue. I agree this is somewhat unexpected. It's not evident, intuitive, and arguably not entirely correct, but the `n` param has to be `>=` `c` param. So as a workaround you can just set the concurrency param `c` to equal the `n` param. For what it's worth, the [docs](https://ghz.sh/docs/options#-n---total) elaborate on this logic slightly, `49 / 50 = 0`, but I admit it's not ideal. As a note I am very slowly reworking the core logic that controls the concurrency, and pacing of requests, which should hopefully make the behaviour better and remove quirks like this one, and a few others that have been brought up in other issues. But my time is pretty limited at the moment, so no concrete ETA, hopefully in near future.

**Comment by bojand** [comment]
Hello, I think this should be resolved in [0.70.0](https://github.com/bojand/ghz/releases/tag/v0.70.0).
